src/scripts/OSFramework/OSUI/Pattern/OverflowMenu/scss/_overflowmenu.scss

```diff
@@ -0,0 +1,57 @@
+/* Patterns - Navigation - OverflowMenu */
+////
+/// @group Patterns-OverflowMenu
+/// Patterns - Navigation - OverflowMenu
+
+.osui-overflow-menu {
+    --border-radius-rounded: 16px;
+
+    &__trigger {
+        --osui-floating-offset: var(--space-s);
+
+        border: var(--border-size-s) solid transparent;
+        border-radius: var(--osui-overflow-menu-shape);
+        color: var(--color-neutral-9);
+        width: 32px;
+
+        * {
+            pointer-events: none;
+        }
+    }
+
+    &--is-open {
+        .osui-overflow-menu__trigger {
+            background-color: var(--color-neutral-4);
+        }
+    }
+    
+    &__balloon {
+        min-width: 170px;
```

Comment:
@BenOsodrac Couldn't we have this as a CSS var? To allow changing it easily and not being a 'magic number'

Reply:
Added it!